docs/_docs/windows.md

```diff
@@ -112,7 +112,7 @@ The development kit did not set the environment path for Ruby so we need to do i
 
  * Open command prompt in `C:\tools\DevKit2`
  * Execute `ruby dk.rb init` to create a file called `config.yml`
- * Edit the `config.yml` file and include the path to Ruby `- C:/tools/ruby22`
+ * Edit the `config.yml` file and include the path to Ruby `- C:/tools/ruby24`
```

Comment:
I don't think this is valid judging by the Ruby version mentioned above (2.2.4).